include/mbedtls/build_info.h

```diff
@@ -74,6 +74,17 @@
 #define MBEDTLS_ARCH_IS_X86
 #endif
 
+/* This is defined if the architecture is Armv8, or higher */
+#if !defined(MBEDTLS_ARCH_IS_ARMV8)
+#if defined(__ARM_ARCH)
+#if __ARM_ARCH >= 8
+#define MBEDTLS_ARCH_IS_ARMV8
```

Comment:
This gets set when building for (e.g.) Cortex-M33. Is that what we want? Or do we want this actually be `MBEDTLS_ARCH_IS_ARMV8A`?

Reply:
That might be better actually - the crypto extensions are Armv8-A only.